Bookish/Controllers/CatalogController.cs

```diff
@@ -14,83 +14,108 @@ public CatalogController(ILogger<CatalogController> logger)
     }
     
     // GET
-    public IActionResult BookEntry(BookInputModel model)
+    public IActionResult BookEntry(DefaultInputModel model)

```

Comment:
It looks like you're adding new data to the database in this method - in which case, it's not a `GET` endpoint